### ISSUE TITLE
OL with bing maps and 2 cluster layers adds 100MB to app runtime

### ISSUE THREAD

**Issue by pke** [openlayers/ol3#4798]
I'd like to use OL in my cordova app and noticed that it adds about 100MB (on Android).

```
const mapLayer = new ol.layer.Tile({
      preload: Infinity,
      source: new ol.source.BingMaps({
        culture: "de-de",
        key: 'XXXXXXXXXXXXXXXXX',
        imagerySet: "Road"
      })
    })

    this.map = new ol.Map({
      view: new ol.View({
        center: ol.proj.fromLonLat([props.center.longitude, props.center.latitude]),
        zoom: props.zoom ? props.zoom : 16
      }),
      loadTilesWhileInteracting: true,
      controls: [],
      logo: false,
      layers: [
        mapLayer,
        retailerLayer,
        garageLayer
      ]
    })
```
I am using 2 cluster layers, where at the moment, only one has 16 icon/text features in it.
The viewport is 320x200 pixels.

**Comment by ahocevar** [comment]
Some browsers unpack images when storing them. In such cases, you may want to compile OpenLayers with a smaller tile cache max size. The default is `ol.DEFAULT_TILE_CACHE_HIGH_WATER_MARK = 2048;`

See #3816 for a related discussion, and #4702 for a suggestion to make the tile cache size configurable in a simpler way.

**Comment by fredj** [comment]
See also #3778

**Comment by fredj** [comment]
Fixed with #4805
